app/views/admin/npq/applications/_application_details.html.erb

```diff
@@ -1,18 +1,33 @@
+<h2 class="govuk-heading-m">Application details</h2>
 <%=
   govuk_summary_list do |sl|
     sl.row do |row|
-      row.key(text: 'Application UUID')
+      row.key(text: 'Application ID')
       row.value(text: application.id || govuk_tag(text: "Missing", colour: "red"))
     end
 
     sl.row do |row|
-      row.key(text: 'User UUID')
+      row.key(text: 'User ID / Participant ID')
       row.value(text: application.user.id || govuk_tag(text: "Missing", colour: "red"))
     end
 
     sl.row do |row|
-      row.key(text: 'User Identity UUID')
-      row.value(text: application.user.id || '-')
+      row.key(text: 'GAI ID')
```

Comment:
@peteryates this is correct in terms of what is considered the GAI ID right?

Reply:
Shouldn't it be `application.user.get_an_identity_id`?